trackingC.Rmd

```diff
@@ -567,6 +567,28 @@ ggplot(plot_data, aes(x = year, fill = def)) +
 
 ```
 
+``` {r af-trend, eval=FALSE}
+trend <- plot_data %>%
+  filter(year %in% 1960:2020)
+
+af_trend <-  trend %>%
```

Comment:
It doesn't look like `trend` is every used for anything...perhaps remove and join these two pipelines together?